src/portable/ehci/ehci.c

```diff
@@ -343,6 +343,8 @@ bool hcd_edpt_open(uint8_t rhport, uint8_t dev_addr, tusb_desc_endpoint_t const
 {
   (void) rhport;
 
+  // Flush cache if required, see hcd.h for details
+  hcd_dcache_flush((uint32_t)ep_desc, sizeof(tusb_desc_endpoint_t));
```

Comment:
do we really need to flush this ? We don't make any transfer yet, would you mind trying without this flush.

Reply:
I will check this, I am still getting to grips with the USB protocol in general and with the code base, I haven't had time to extensively study the code yet, so please bare with me.  😊

Reply:
As above...
> I will check this, I am still getting to grips with the USB protocol in general and with the code base, I haven't had time to extensively study the code yet, so please bare with me. 😊

Reply:
Hi @hathach, 

Yes this is my bad don't know what I was thinking 😊